docs/src/tutorials/conic/min_ellipse.jl

```diff
@@ -3,48 +3,101 @@
+
+# It is convenient to parameterize as the ellipse `{ x : || Px + q || ≤ 1 }`.
+
+# Then the optimal `P` and `q` are given by the convex program
+
+# ````
+#       maximize   log(det(P))
+      
+#     subject to   t[i] ≥ 0, i = 1 ... m
+    
+#                  [ P^2 - t[i] A[i]   P q - t[i] b[i]       0
+#                    P q - t[i] b[i]    -1 - t[i] c[i]    (P q)'
+#                                 0              (P q)   - P^2   ]    PSD, i = 1 ... m
+# ````
+# with helper variables `t[i]`. 
 
 using JuMP
-import LinearAlgebra
-import SCS
-import Test
-
-function example_min_ellipse()
-    ## We will use three ellipses: two "simple" ones, and a random one.
-    As = [
-        [2.0 0.0; 0.0 1.0],
-        [1.0 0.0; 0.0 3.0],
-        [2.86715 1.60645; 1.60645 1.12639],
+using SCS
+using Plots
+
+# Input ellipses, parameterized as
+# x' A x + 2 b' x + c ≤ 0
+As = [
+        [1.2576 -0.3873; -0.3873 0.3467],
+        [1.4125 -2.1777; -2.1777 6.7775],
+        [1.7018  0.8141;  0.8141 1.7538],
+        [0.9742 -0.7202; -0.7202 1.5444],
+        [0.6798 -0.1424; -0.1424 0.6871],
+        [0.1796 -0.1423; -0.1423 2.6181]
+    ]
+bs = [
+        [ 0.2722,  0.1969],
+        [-1.228 , -0.0521],
+        [-0.4049,  1.5713],
+        [ 0.0265,  0.5623],
+        [-0.4301, -1.0157],
+        [-0.3286,  0.557 ]
     ]
-    ## We change the weights to see different solutions, if they exist
-    weights = [1.0 0.0; 0.0 1.0]
-    model = Model(SCS.Optimizer)
-    set_silent(model)
-    @variable(model, X[i = 1:2, j = 1:2], PSD)
-    @objective(model, Min, LinearAlgebra.tr(weights * X))
-    @constraint(model, [As_i in As], X >= As_i, PSDCone())
-    optimize!(model)
-    Test.@test termination_status(model) == OPTIMAL
-    Test.@test primal_status(model) == FEASIBLE_POINT
-    Test.@test objective_value(model) ≈ 6.46236 atol = 1e-4
-    Test.@test value.(X) ≈ [3.1651 0.8022; 0.8022 3.2972] atol = 1e-4
-    return
+cs = [0.1831, 0.3295, 0.2077, 0.2362, 0.3284, 0.4931]
+
+function minimal_ellipse(As, bs, cs)
+    # Build the model under the change of variables
+    #       Psqr = P^2
+    #    q_tilde = Pq
+    mdl = Model(SCS.Optimizer)
+
+    m = length(As)
+    n, _ = size(first(As))
+
+    @variable(mdl, Psqr[1:n, 1:n], PSD)
+    @variable(mdl, tau[1:m] ≥ 0)
+    @variable(mdl, q_tilde[1:n])
+    @variable(mdl, logdetP)
+    @constraint(mdl, [logdetP; [Psqr[i, j] for i in 1:n for j in i:n]] in MOI.RootDetConeTriangle(n))
+
+    for (A, b, c, t) in zip(As, bs, cs, tau)
+        @constraint(mdl,
+            -[      Psqr - t*A     q_tilde - t*b    zeros(n,n)  ;
+                (q_tilde - t*b)'        -1 - t*c      q_tilde'  ;
+                     zeros(n,n)          q_tilde        -Psqr    ]
+            in PSDCone())
+    end
+
+    @objective(mdl, Max, logdetP)
+
+    optimize!(mdl)
+
+    @assert termination_status(mdl) == OPTIMAL
+    @assert primal_status(mdl) == FEASIBLE_POINT
+    @assert isapprox(objective_value(mdl), 0.132413, atol=1e-4)
```

Comment:
I think this last `@assert` shouldn't be inside the function?

Reply:
Nice catch, will fix shortly.